codecov.yml

```diff
@@ -0,0 +1,14 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 3%
+        base: auto 
```

Comment:
```
base
[Deprecated as of July 2020 -- Codecov's product roadmap includes a novel way to manually pick base of pull requests]
```

According to https://docs.codecov.com/docs/commit-status#base, this parameter seems unnecessary.